db/commitlog/commitlog.cc

```diff
@@ -2613,7 +2621,7 @@ db::commitlog::read_log_file(sstring filename, sstring pfx, commit_load_reader_f
                 // zero length file. accept it just to be nice.
                 co_return;
             }
-            // Will throw if we got eof
+            // Will throw if we got eof--
```

Comment:
What's the significance of eof--? Please be less cryptic.

Reply:
I think that is an honest typo. Weird though.